.terraform.lock.hcl

```diff
@@ -6,7 +6,6 @@ provider "registry.terraform.io/chainguard-dev/apko" {
   constraints = "0.15.2"
   hashes = [
     "h1:ZAxVcQUI8ZRDiYDd599YQPAyPhb/9Mj28EUjsmEL0oc=",
-    "h1:pDYdU+HO6RchUNnGKiCVsp+3XFFS35Bfqre5mwTZ0Vg=",
     "zh:29d2d68a8c49c216e3f29070bdb2bc259420b6b4074f3c9300540b2add567573",
```

Comment:
terraform.lock file updates should be submitted separately, and they should not delete hashes.